### PR TITLE
Adding Info About Organization Scope

### PR DESCRIPTION
**Pull request by bentleycook**
Adding a bit of info regarding storing data at the organization scope.

This was in the glitch project comments, but not the docs.

### CODE REVIEW

File: app/templates/power-ups/topics.html [hunk 1]
```diff
@@ -183,6 +183,7 @@ <h2 id="data-storage">Data Storage</h2>
   <pre>t.set('board', 'private', 'key', value)</pre>
   <p>or</p>
   <pre>t.set('board', 'private', { key1: value1, key2: value2 })</pre>
+  <p>Storing at the 'organization' scope will only work if the active user is a member of the team to which the board belongs.</p>
```
Comment (mpcowan) [inline app/templates/power-ups/topics.html]: Maybe word it as:

`Storing at the 'organization' scope will only work if the both the board is in a team and the active user is a member of the team to which the board belongs.`

Comment (mpcowan) [inline app/templates/power-ups/topics.html]: Just to make clear that setting data at the organization level doesn't work for personal boards that have no organization.

Comment (bentleycook) [inline app/templates/power-ups/topics.html]: Good call. Updated and added a bit about recommended fall-back for when organization fails.

File: app/templates/power-ups/topics.html [hunk 2]
```diff
@@ -183,8 +183,14 @@ <h2 id="data-storage">Data Storage</h2>
...
-  <p>Currently, there is no external API for accessing Power-Up data. Power-Up stored data can only be retrieved when the UI
-    is loaded by the member with the Power-Up enabled.</p>
+  <p>Storing at the 'organization' scope will only work if the both the board is in a team and the active user is a member of the team to which the board belongs. Because the 'organization' scope is not always available, you should add a catch whenever trying to set data at the 'organization' level and fallback to the 'board' level, if necessary. For example:</p>
+<pre>t.set('organization', 'private', 'key', 'value')
+  .catch(t.NotHandled, function() {
+    // fall back to storing at board level
+    return t.set('board', 'private', 'key', 'value');
+});</pre>
+  
+  <p>Currently, there is no external API for accessing Power-Up data. Power-Up stored data can only be retrieved when the UI is loaded by the member with the Power-Up enabled.</p>
```
Comment (mpcowan) [inline app/templates/power-ups/topics.html]: This statement is no longer true. You can read 'shared' pluginData via the API.

File: app/templates/power-ups/topics.html [hunk 3]
```diff
@@ -183,9 +183,14 @@ <h2 id="data-storage">Data Storage</h2>
...
-    is loaded by the member with the Power-Up enabled.</p>
+  <p>Storing at the 'organization' scope will only work if the both the board is in a team and the active user is a member of the team to which the board belongs. Because the 'organization' scope is not always available, you should add a catch whenever trying to set data at the 'organization' level and fallback to the 'board' level, if necessary. For example:</p>
+<pre>t.set('organization', 'private', 'key', 'value')
+  .catch(t.NotHandled, function() {
+    // fall back to storing at board level
+    return t.set('board', 'private', 'key', 'value');
+});</pre>
   <p>Members can reset their private data at any time via the Power-Ups menu.</p>
+  <p>Shared Power-Up data is available via the API at the organization, board, and card level. Each resource returns the data (named pluginData) that was set at that scope. For instance, `1/card/IdCard/pluginData` will return data set with `t.set('card', 'shared', 'key', 'value')`. Currently, pluginData is read-only and can not be set via the API. Data set with within the private scope is not available via the API.</p>
```
Comment (mpcowan) [inline app/templates/power-ups/topics.html]: Slight typo towards the end:

> Data set with within the private scope is not available via the API.

**Pull request merged**